package.json

```diff
@@ -4,9 +4,9 @@
   "description": "A responsive and accessible date range picker component built with React",
   "main": "index.js",
   "scripts": {
-    "build": "npm run clean && npm run build:js && npm run build:css && npm run build:svg",
+    "build": "npm run clean && npm run build:js && npm run build:svg && mkdir lib/css && npm run build:css -- --optimize ",
```

Comment:
should `--optimize` just go in build:css? or is it useful to have it separate?

also, the `mkdir -p` should go in `prebuild:css` instead, and we should also have a script that rimrafs lib/css prior to building, just in case

Reply:
`--optimize` is quite slow (like pretty damn slow), so I wanted a fast version of `build:css` that would run as part of the `storybook:css` script.